### PR TITLE
83 link each tool to the arc projects that use it or vice versa

### LINKED ISSUE #83: Link each tool to the ARC projects that use it (or vice versa?)

### PR DESCRIPTION
**Pull request by katiebuntic**
- updating tables to link projects with each tool

### CODE REVIEW

File: docs/pages/templates.md [hunk 1]
```diff
@@ -29,10 +29,10 @@ If you're making a package within one of these communities, we recommend using t
...
+| Name                                                                                                                          | Short description                                                                                                            | 🚦  | Used by |
+| ----------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- | ----- | :-: |
+| [Cookiecutter](https://github.com/cookiecutter/cookiecutter)                                                                  | Widely-used tool for creating project templates, but is not maintained.                                                      | 🔴  | |
+| [Cookieninja](https://github.com/cookieninja-generator/cookieninja)                                                           | A fork of Cookiecutter that is actively maintained.                                                                          | 🟢  | |
```
Comment (samcunliffe) [inline docs/pages/templates.md]:
```suggestion
| [Cookiecutter](https://github.com/cookiecutter/cookiecutter)                                                                  | Widely-used tool for creating project templates.                                                      | 🟢   | |
| [Cookieninja](https://github.com/cookieninja-generator/cookieninja)                                                           | A fork of Cookiecutter that is not actively maintained.                                                                          | 🔴  | |
```

File: docs/pages/docs.md [hunk 2]
```diff
@@ -10,8 +10,8 @@ layout: default
 | Name                                            | Short description                                               | 🚦  | Used by                                                                |
 | ----------------------------------------------- | --------------------------------------------------------------- | :-: | ---------------------------------------------------------------------- |
 | [sphinx](https://www.sphinx-doc.org/en/master/) | Generates documentation from reStructuredText or Markdown files | 🟢  | [WAZP](https://github.com/SainsburyWellcomeCentre/WAZP/tree/main/docs) |
-| [mkdocs](https://www.mkdocs.org/)               | Generates documentation from Markdown files                     | 🟠  |                                                                        |
-| [pdoc](https://pdoc.dev/)                       | Auto-generates API documentation from docstrings                | 🟠  | [sleplet](https://github.com/astro-informatics/sleplet/)               |
```
Comment (paddyroddy) [inline docs/pages/docs.md:14]: Not sure why you've removed this, can assure you it is using `pdoc3`

File: docs/pages/templates.md (hunk 1 above)
Comment (paddyroddy) [reply docs/pages/templates.md]: Via #186

**Review by paddyroddy — CHANGES_REQUESTED** [review]

**Comment by paddyroddy** [comment]
Have fixed linting for you & updated from `main`

**Comment by paddyroddy** [comment]
@samcunliffe the link checker CI is failing, and I can't work out why. Any ideas?

**Comment by samcunliffe** [comment]
[https://github.com/UCL-MIRSG/UCLH-MPBE-SRR-XNAT](https://github.com/UCL-MIRSG/UCLH-MPBE-SRR-XNAT](https://github.com/UCL-MIRSG/UCLH-MPBE-SRR-XNAT)) is 404 unless you're a member of UCL-MIRSG which we're not all. And TREx is private to UCL-ARC.

**Comment by paddyroddy** [comment]
> [https://github.com/UCL-MIRSG/UCLH-MPBE-SRR-XNAT](https://github.com/UCL-MIRSG/UCLH-MPBE-SRR-XNAT%5D(https://github.com/UCL-MIRSG/UCLH-MPBE-SRR-XNAT)) is 404 unless you're a member of UCL-MIRSG which we're not all. And TREx is private to UCL-ARC.

Oh yeah of course! Didn't think of that as I'm part of MIRSG. @katiebuntic you'll need to use public repos generally

**Review by samcunliffe — APPROVED** [review]
Looks good to me!
@paddyroddy are you happy?

**Review by paddyroddy — APPROVED** [review]

**Review by paddyroddy — CHANGES_REQUESTED** [review]
hand on clicked the wrong button

File: docs/Gemfile.lock [hunk 3]
```diff
@@ -26,7 +26,7 @@ GEM
       liquid (~> 4.0)
       mercenary (>= 0.3.6, < 0.5)
       pathutil (~> 0.9)
-      rouge (>= 3.0, < 5.0)
+      rogue (>= 3.0, < 5.0)
```
Comment (paddyroddy) [inline docs/Gemfile.lock]: Do we know why this has changed? Is this an actual change or `codespell`?

**Review by paddyroddy — CHANGES_REQUESTED** [review]
- I've merged in `main`
- I've fixed the table formatting issues in `templates.md`
- I've moved `cookieninja` to 🔴 and `cookiecutter` to 🟢

`Requesting changes` due to the `Gemfile` thing

File: docs/pages/templates.md (hunk 1 above)
Comment (paddyroddy) [reply docs/pages/templates.md]: This didn't get applied, but I did it manually

File: docs/Gemfile.lock (hunk 3 above)
Comment (samcunliffe) [reply docs/Gemfile.lock]: 😱 I'm going blind: I thought this was just whitespace.
Good catch.

Yeah I'm 👎  on `codespell` in pre-commit. I didn't actually realise we had added it.

Comment (paddyroddy) [reply docs/Gemfile.lock]: Fixing now!

Comment (paddyroddy) [reply docs/Gemfile.lock]: And it was @dstansby

Comment (paddyroddy) [reply docs/Gemfile.lock]: #207

**Review by paddyroddy — APPROVED** [review]
I've fixed the `Gemfile` issue. So happy to merge assuming CI passes @samcunliffe

**Pull request merged**